src/main/webapp/resources/js/pages/projects/samples/components/SamplesMenu.jsx

```diff
@@ -144,9 +147,31 @@ export default function SamplesMenu() {
    */
   const shareSamples = () => {
     if (selected.size === 0) return;
-    formatAndStoreSamples(`share`);
-    // Redirect user to share page
-    window.location.href = setBaseUrl(`/projects/${projectId}/share`);
+    const { valid, locked } = validateSamplesForMergeOrShare(selected);
+    if (locked.length) {
+      if (valid.length === 0) {
+        message.error(i18n("SampleMenu.share-all-locked"));
```

Comment:
Can we update this to use notifcation instead of message since that's what we use in other places

Reply:
Sure, updated in 80fb51ffff3350dc8955d65e5a8601747345ab7c